diff --git a/Nickvision.Parabolic.Shared/Services/YtdlpExecutableService.cs b/Nickvision.Parabolic.Shared/Services/YtdlpExecutableService.cs
index 3fd2d78..1871660 100644
--- a/Nickvision.Parabolic.Shared/Services/YtdlpExecutableService.cs
+++ b/Nickvision.Parabolic.Shared/Services/YtdlpExecutableService.cs
@@ -97,8 +97,13 @@ public class YtdlpExecutableService : IYtdlpExecutableService
             {
                 using var process = new Process()
                 {
-                    StartInfo = new ProcessStartInfo("chmod", [$"+x \"{path}\""])
+                    StartInfo = new ProcessStartInfo
                     {
+                        FileName = "chmod",
+                        ArgumentList = {
+                          "0755",
+                          path,
+                        },
                         UseShellExecute = false,
                         CreateNoWindow = true,
                     }
